buildcloud/build_cloud.py

```diff
@@ -71,7 +72,12 @@ def env(args):
 
         new_names = []
         for model in args.model:
-            name = rename_env(model, 'cwr-', os.path.join(
+            prefix = 'cwr-'
+            if 'azure' in model.lower():
+                # Use Jenkins BUILD_NUMBER if it is available as a unique name.
```

Comment:
Does this address a specific bug or issue? If so, it would be nice to have a link to that bug in the comment, so that we can get the history of the issue, if we ever decide to change this check.